input/fsh/examples/cinc/CommunicationRequestEmailExample.fsh

```diff
@@ -0,0 +1,74 @@
+Instance : EmailCommunicationRequestExample
+InstanceOf : CMSCommunicationRequest
+Description: "An example of a CommunicationRequest resource. This resource is used to record information related to requesting communication with a client using EMAIL as the medium."
+Usage : #example
+* identifier[0].use = #usual
+* identifier[0].value = "#ebdb8d66-1390-42c7-89a6-a9a75d65d0e6"
+* identifier[0].system = "https://api.messaging.digital.health.nz/ConsumerRecordId"
+* identifier[1].use = #secondary
+* identifier[1].system = "https://api.messaging.digital.health.nz/CorrelationID"
+* identifier[1].value = "#bdb8d66-1390-42c7-89a6-a9a75d65d0e6-001"
+* identifier[2].use = #secondary
+* identifier[2].value = "#jb8d66-1390-42c7-89a6-a9a75d65d344"
+* identifier[2].system = "https://api.messaging.digital.health.nz/MessageID"
+* identifier[3].use = #secondary
+* identifier[3].value = "#kb8d66-1390-42c7-89a6-a9a75d65d333"
+* identifier[3].system = "https://api.messaging.digital.health.nz/ProviderMessageID"
+* identifier[4].use = #secondary
+* identifier[4].value = "#lb8d66-1390-42c7-89a6-a9a75d65d333"
+* identifier[4].system = "https://api.messaging.digital.health.nz/ConsumerID"
+* status = #unknown
+* medium.coding.code = #MAILWRIT
+* medium.coding.system = "http://terminology.hl7.org/CodeSystem/v3-ParticipationMode"
+* occurrenceDateTime = "2024-08-20T14:30:00Z"
+* authoredOn = "2024-08-20T14:30:00Z"
+* recipient.type = "Practitioner"
+* recipient.identifier.use = #official
+* recipient.identifier.system = "https://standards.digital.health.nz/ns/hpi-person-id"
+* recipient.identifier.value = "99ZZFX"
+* subject.identifier.use = #official
+* subject.identifier.system = "https://standards.digital.health.nz/ns/nhi-id"
+* subject.identifier.value = "ZXP7823"
+* subject.display = "Carey Carrington"
+* sender = Reference(BSAService)
+//Email Body
+* payload[0].contentAttachment.contentType = #application/pdf
+* payload[=].contentAttachment.language = #en-nz
+* payload[=].contentAttachment.id = "attachment-body"
+* payload[=].contentAttachment.title = "Appointment Reminder" //Email Subject
+* payload[=].contentAttachment.creation = "2023-08-01T09:35:00+11:00"
+* payload[=].contentAttachment.data = "IyBQYXRpZW50IEVuY291bnRlciBOb3RlcwoKIyMgMjAyMy0wOC0wMgoKLSBEaXNjdXNzZWQgY29uZGl0aW9uCi0gT3RhaW5lZCBjb25zZW50IAotIGV0Yy4="
+
+//Attachment1
+* payload[+].contentAttachment.contentType = #application/pdf
+* payload[=].contentAttachment.id = "attcahment-file1"
```

Comment:
Typo in ID -- and, how are these inline file attachments going to be distinguished from attachments like "attachment-body" (if you're allowing non-templated emails to be passed inline) and "attachment-template"? Are you going to have a few special named IDs which get handled differently, and everything else gets passed through as an attachment, or are you going to have certain contentTypes which always get passed through as attachments?